frb_spectral_search/spectral_search.py

```diff
@@ -270,7 +269,7 @@ def calculate_noise(ww, time_range, on_range, valid_channels, sig1, sig2, sig3,
         if x < 1:
             scale2 = np.append(scale2, 1)
         else:
-            scale2 = np.append(scale2, x)          
+            scale2 = np.append(scale2, x)
     if scale:
         scale_fact = np.nanmean(scale2)
```

Comment:
I think lines 267-273 is equivalent to `scale2 = np.minimum(scale_fact_calc,np.ones(scale_fact_calc.size))`, which takes the element-wise minimum (giving 1 if `scale_fact_calc` is smaller than 1, and `scale_fact_calc` otherwise).